pkg/proxy/ipvs/README.md

```diff
@@ -334,31 +334,8 @@ export KUBE_PROXY_MODE=ipvs
 
 ### Cluster Created by Kubeadm
 
-If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you have to add `mode: ipvs` and also add `SupportIPVSProxyMode: true` below the `kubeProxy` field as part of the kubeadm configuration.
```

Comment:
This part you can remove the reference to the feature flag, but keep the 'mode: ipvs' as this is still true